cpp/merge_differential_and_full_marc_updates_merge_superior.sh

```diff
@@ -59,10 +59,15 @@ function CreateArchiveWithMergedTitles() {
 
 
 # Argument processing
-KEEP_ITERMEDIATE_FILES=
+KEEP_INTERMEDIATE_FILES=
+SKIP_DB_UPDATES=
 if [[ $# > 1 ]]; then
     if [[ $1 == "--keep-intermediate-files" ]]; then
-        KEEP_ITERMEDIATE_FILES="--keep-intermediate-files"
+        KEEP_INTERMEDIATE_FILES="--keep-intermediate-files"
+        shift
+    fi
+    if [[ $1 == "--skip-db-updates" ]]; then
```

Comment:
This code looks like you can only specify --keep-intermediate-files or --skip-db-updates o/w you would have to use a loop here.  If I am correct the usage message should reflect this and look like [(--keep-intermediate-files|--skip-db-updates)]

Reply:
"shift" should adjust the positional parameters so both can be used at the same time.

Reply:
While that is true, what happens when you call the script with only the --keep-intermediate-files flag and an email address?  Don't you run into line 72 even though the --skip-db-updates flag is optional?